CONTRIBUTING.md

```diff
@@ -50,17 +50,17 @@ But what do we do when waiting for merge into master \(wanting to keep PRs small
   * but what about when you are waiting for merge?
   * solutions
     * 1\) put 2nd PR into branch that the first PR is hitting - but requires update after merging
-    * 2\) prefer to leave exiting PR until it can be reviewed, and instead go and work on some other part of the codebase that is not impacted by the first PR
+    * 2\) prefer to leave existing PR until it can be reviewed, and instead go and work on some other part of the codebase that is not impacted by the first PR
 
 ### Code Review
 * Github setting in place - at least one review is required to merge
   - in principle anyone (who is not the PR owner) can review
-  - but often it will be the core developers (Robert, Ulf, Greg, Wolfgang?)
+  - but often it will be the core developers (Robert, Wolfgang, Matt, Alina, Alex)
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-25T08:49:36Z" title="Wednesday, September 25th 2019, 10:49:36 am +02:00">Sep 25, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -50,17 +50,17 @@ But what do we do when waiting for merge into master \(wanting to keep PRs small
   * but what about when you are waiting for merge?
   * solutions
     * 1\) put 2nd PR into branch that the first PR is hitting - but requires update after merging
-    * 2\) prefer to leave exiting PR until it can be reviewed, and instead go and work on some other part of the codebase that is not impacted by the first PR
+    * 2\) prefer to leave existing PR until it can be reviewed, and instead go and work on some other part of the codebase that is not impacted by the first PR
 
 ### Code Review
 * Github setting in place - at least one review is required to merge
   - in principle anyone (who is not the PR owner) can review
-  - but often it will be the core developers (Robert, Ulf, Greg, Wolfgang?)
+  - but often it will be the core developers (Robert, Wolfgang, Matt, Alina, Alex)
```


😘